tests/test_metadata/test_consolidated.py

```diff
@@ -573,3 +573,38 @@ async def test_use_consolidated_false(
             assert len([x async for x in good.members()]) == 2
             assert good.metadata.consolidated_metadata
             assert sorted(good.metadata.consolidated_metadata.metadata) == ["a", "b"]
+
+
+@pytest.mark.parametrize("zarr_format", [2, 3])
```

Comment:
```suggestion
```
we have a zarr format fixture, so I think this explicit parametrization is not needed